package.json

```diff
@@ -1,27 +1,57 @@
 {
-  "name": "xnft-quickstart",
-  "version": "2.0.0",
-  "description": "",
-  "main": "index.js",
-  "author": "",
-  "license": "ISC",
+  "main": "node_modules/expo/AppEntry.js",
   "scripts": {
-    "build": "npx xnft native build",
     "start": "npx xnft native start",
-    "dev": "npx xnft native dev"
-  },
-  "devDependencies": {
-    "@coral-xyz/xnft-cli": "latest"
+    "build": "expo export:web",
+    "dev": "expo start --web & npx xnft dev --iframe http://localhost:19006"
   },
   "dependencies": {
-    "@coral-xyz/common-public": "latest",
-    "@types/react-native": "^0.70.6",
+    "@coral-xyz/common-public": "^0.2.0-latest.971",
+    "@expo-google-fonts/dev": "*",
+    "@expo/vector-icons": "^13.0.0",
+    "@expo/webpack-config": "^0.17.2",
+    "@react-native-community/masked-view": "*",
+    "@react-navigation/bottom-tabs": "6.3.1",
+    "@react-navigation/drawer": "6.4.1",
+    "@react-navigation/elements": "1.3.3",
+    "@react-navigation/material-bottom-tabs": "6.2.1",
+    "@react-navigation/material-top-tabs": "6.2.1",
+    "@react-navigation/native": "6.0.10",
+    "@react-navigation/native-stack": "6.6.1",
+    "@react-navigation/stack": "6.2.1",
+    "@solana/web3.js": "^1.67.2",
+    "expo": "~47.0.8",
+    "expo-constants": "~14.0.2",
+    "expo-linking": "~3.2.3",
+    "expo-status-bar": "~1.4.2",
     "react": "18.1.0",
     "react-dom": "18.1.0",
     "react-native": "0.70.5",
-    "react-native-web": "~0.18.7",
-    "react-xnft": "latest",
-    "recoil": "0.7.5",
-    "twrnc": "^3.4.1"
+    "react-native-gesture-handler": "~2.8.0",
+    "react-native-pager-view": "6.0.1",
+    "react-native-paper": "^4.7.2",
+    "react-native-reanimated": "~2.12.0",
+    "react-native-safe-area-context": "4.4.1",
+    "react-native-screens": "~3.18.0",
+    "react-native-tab-view": "^3.0.0",
+    "react-native-web": "~0.18.9",
+    "react-error-overlay": "6.0.9",
+    "recoil": "*",
+    "twrnc": "*",
+    "uuid": "*"
+  },
+  "devDependencies": {
+    "xnft": "latest",
+    "@babel/core": "^7.12.9",
+    "@types/react": "~18.0.14",
+    "@types/react-native": "~0.70.6",
+    "typescript": "^4.6.3"
+  },
+  "resolutions": {
+    "react-error-overlay": "6.0.9"
+  },
+  "private": true,
+  "engines": {
+    "node": "<17"
```

Comment:
i think this is because you're using the old expo cli that's going to get removed in the next expo version. you're getting an error that says node v18 doesn't work right?

Reply:
yea it throws an ERR_OSSL_EVP_UNSUPPORTED error